less/navigation.less

```
@@ -1 +1,29 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.navigation{
+    position: fixed;
+    width: 100%;
+    opacity: .8;
+    .flexed;
+    flex-direction: column;
+    border-bottom: @nav-border;
+    background: @nav-bg;
+    opacity: 25%;
+    .logo{
+        text-align:center;
+        margin: 1rem 0 1rem 0;
+    }
+    nav{
+        .flexed;
+        .centered;
+        width:45%;
+        justify-content: space-around;
+        @media @phone{
+            width: 80%;
+        }
+    }
+        a{
+            margin: 1rem 0rem 1rem 0rem;
```

```suggestion
            margin: 1% 0 1% 0;
```
Typically only want to use rems for font-size, as depending on the user's browser setting it can really mess up your styles otherwise.